idea/testData/inspectionsLocal/branched/introduceWhenSubject/qualified.kt

```diff
@@ -0,0 +1,13 @@
+import Platform.JvmPlatform
+
+sealed class Platform {
+    object JvmPlatform : Platform()
+    class Another(val name: String) : Platform()
+}
+
+class ModuleInfo(val platform: Platform)
+
+fun foo(moduleInfo: ModuleInfo) = <caret>when {
+    moduleInfo.platform == JvmPlatform -> 1
```

Comment:
Please check how it works if the condition is flipped: `JvmPlatform == moduleInfo.platform`.